.gitmodules

```diff
@@ -10,3 +10,6 @@
 	path = clang
 	url = https://github.com/RadeonOpenCompute/hcc-clang-upgrade.git
 	branch = clang_tot_upgrade
+[submodule "compiler-rt"]
```

Comment:
updated the Jenkinsfile...I had this in my mind but so how still forgot to update it :)